src/accountSelector.tsx

```diff
@@ -13,7 +13,7 @@ import {
 
 import { useSubstrate } from './substrate-lib';
 
-function Main (props) {
+function Main (props: any) {
```

Comment:
any should be avoided and strong typing should be used

Reply:
Consider doing the same you do on another props like that.
```suggestion
function Main ({ setAccountAddress }: any) {
```

In place of any put some type like MainPropsType or AccountSelectorType.